aws/resource_aws_ecs_capacity_provider.go

```diff
@@ -63,6 +63,13 @@ func resourceAwsEcsCapacityProvider() *schema.Resource {
 							ForceNew: true,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
+									"instance_warmup_period": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										Computed:     true,
+										ForceNew:     true,
+										ValidateFunc: validation.IntAtLeast(1),
```

Comment:
Nit: [API Reference](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ManagedScaling.html#ECS-Type-ManagedScaling-instanceWarmupPeriod) denotes maximum of 10000

```suggestion
										ValidateFunc: validation.IntBetween(1, 10000),
```